catalog.yml

```diff
@@ -0,0 +1,30 @@
+metadata:
+  version: 1
+sources: 
+  # processed data - intermediate 
+  mills_official:
+    driver: csv 
+    description: list of offical mills act agencies, result of controller parse.csv 
+    args:
+      urlpath: "./intermediate/mills_official.csv"
+  gsheet_trillium_list: 
+    driver: csv
+    description: list from the trillium master and filtered. download using the bash script. 
+    args:
+      urlpath: "./intermediate/offical_list_from_gsheet.csv"
+  ctsa_list: 
+    driver: csv
+    description: the CTSA list of paratransit agencies, processed from the wedny king email. 
```

Comment:
would you annotate with dates files published or received as appropriate? ie date of email file was _sent by King._

Reply:
updated!